src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs

```diff
@@ -932,7 +932,7 @@ static void CheckKeepAlive(object ctx)
             TimeSpan elapsedSinceLastActive = DateTime.UtcNow - self.lastClientActivityTime;
             if (elapsedSinceLastActive > self.keepAliveTimeout)
             {
-                ShutdownOnError(context, string.Empty, new ProtocolGatewayException(ErrorCode.KeepAliveTimedOut, "Keep Alive timed out."));
+                ShutdownOnError(context, string.Empty, new ProtocolGatewayException(ErrorCode.KeepAliveTimedOut, $"Keep Alive timed out {self.keepAliveTimeout}."));
```

Comment:
$"Keep Alive timed out. Timeout: {self.keepAliveTimeout}"

Reply:
I will modify the log string